Sources/LiveViewNative/LiveView.swift

```diff
@@ -66,6 +63,10 @@ public struct LiveView<R: RootRegistry>: View {
     public init(url: URL, configuration: LiveSessionConfiguration = .init()) {
         self.init(session: .init(url, config: configuration))
     }
+    
+    private var rootCoordinator: LiveViewCoordinator<R> {
+        session.navigationPath.first!.coordinator
+    }
```

Comment:
The `rootCoordinator` is now a computed property. This allows `redirect` navigation to be performed from the root of the app.